packages/genesys-spark/src/hosts.ts

```diff
@@ -51,5 +44,14 @@ export function getFontOrigin(): string {
 
 function getRegionDomain() {
   const pageHost = window.location.hostname;
-  return DOMAIN_LIST.find(regionDomain => pageHost.endsWith(regionDomain));
+
+  // We can automatically handle the standard domain format: ${region}.pure.cloud
+  if (pageHost.endsWith('.pure.cloud')) {
+    return pageHost.split('.').slice(-3).join('.');
```

Comment:
I'm a bit confused by this. From my understanding, if the pageHost is something like 'aps1.pure.cloud', wouldn't this just also return 'aps1.pure.cloud' in a very roundabout way? Is there a case where there could be more than 2 periods? I don't see a current existing domain from the ones you deleted that do.

Also the `pageHost.endsWith` will not match on of the domains deleted above: `use2.maximus-pure.cloud`

Reply:
use2.maximus-pure.cloud is no longer in use, so it doesn't need a replacement. I should have called that out in the PR.

`pageHost` will normally be 4 segments long. Our apps are hosted on a subdomain, eg. `apps.aps1.pure.cloud` or `app.aps.pure.cloud`. Some projects could create new sub-domains as well. We want all of these and as many future variations as possible to resolve back to just the region portion - the last 3 segments.